### PR TITLE
doc: updated OS recommendations and distro list

### PR DESCRIPTION
**Pull request by capri1989**
I updated the OS recommendations page to reflect Nautilus and I also added openSUSE to the list of distros there. I also added openSUSE to the "Get Packages" page with the required links. 

Signed-off-by: Kai Wagner <kwagner@suse.com>

### CODE REVIEW

File: doc/start/os-recommendations.rst [hunk 1]
```diff
@@ -39,6 +39,31 @@ platforms.  Generally speaking, there is very little dependence on
 specific distributions aside from the kernel and system initialization
 package (i.e., sysvinit, upstart, systemd).
 
+Nautilus (14.2.z)
```
Comment (sebastian-philipp) [inline doc/start/os-recommendations.rst]: Would it make sense to clean up the tables for EOL releases from this page as well?

Comment (capri1989) [reply doc/start/os-recommendations.rst]: Sure I think that would make sense

File: doc/install/get-packages.rst [hunk 2]
```diff
@@ -185,6 +188,19 @@ You can download the RPMs directly from::
...
+
+You need to add the Ceph package repository to your list of zypper sources. This can be done with the following command ::
+
+    zypper ar https://download.opensuse.org/repositories/filesystems:/ceph/openSUSE_Leap_15.1/filesystems:ceph.repo
+
+openSUSE Tumbleweed
+-------------------
+
+The newest major release of Ceph is already available through the normal Tumbleweed repositories.
+There's no need to manually add another repo.
```
Comment (LenzGr) [inline doc/install/get-packages.rst]:
```suggestion
There's no need to manually add another package repository. 
```

**Comment by capri1989** [comment]
> Please prefix the title of your commit message with the subcomponent you are changing ("doc: "). See also [1]. This is required for your PR to be merged.
> 
> [1] https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes



> Please prefix the title of your commit message with the subcomponent you are changing ("doc: "). See also [1]. This is required for your PR to be merged.
> 
> [1] https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes

done

**Comment by capri1989** [comment]
@batrick Could that one be merged?

**Pull request merged**